pyproject.toml

```diff
@@ -31,6 +31,7 @@ dependencies = [
     "rasterio",
     "requests",
     "rioxarray",
+    "ruff",
```

Comment:
ruff is now twice in the list.
I didn't review this but aren't ruff and pre-commit more developpers dependencies? So to exclude from a conda-forge install?
I also see there is no real 'developper' or dev install with basically all+test+doc

Reply:
You're right about it being in there twice, that was my mistake. As for the second part of your comment I think you're correct as well. Previously there wasn't really such an option. I'll try to restructure them a bit to account for this